dotcom-rendering/src/web/components/ElementContainer.tsx

```diff
@@ -39,7 +39,15 @@ type Props = {
 	borderColour?: string;
 	children?: React.ReactNode;
 	shouldCenter?: boolean;
-	element?: 'div' | 'article' | 'aside' | 'nav'; // ElementContainer is generally a top-level wrapper
+	element?:
+		| 'div'
+		| 'article'
+		| 'aside'
+		| 'nav'
+		| 'main'
```

Comment:
I think `main` wasn't added because there wasn't an expected use case for it but I think this shows there is so we could think about adding it.

At some point we need to consider how we rationalise ElementContainer with the source version.